rtf_vis_tool/.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+.eslintcache
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
Just a `.gitignore` that's auto generated when I first initialized the react application.

Reply:
can these be placed in the main .gitignore file, if we just put `*/` in front of each one?
e.g.
*/npm-debug.log*
?